packages/adapters/bitcoin/src/adapter.ts

```diff
@@ -207,6 +224,40 @@ export class BitcoinAdapter extends AdapterBlueprint<BitcoinConnector> {
     // Get capabilities
     return Promise.resolve('0x')
   }
+
+  // -- Private ------------------------------------------ //
+  private bindEvents(connector: BitcoinConnector) {
+    this.unbindEvents()
```

Comment:
should when disconnecting as well ?

Reply:
https://github.com/reown-com/appkit/pull/3363/files#diff-1f24511c3a1a14334ac74aee065ecfe3792cd10bc420448b63e6b95bb1ff6be3R85